test/DynamoCoreTests/NodeToCodeTest.cs

```diff
@@ -1241,12 +1241,13 @@ protected override void GetLibrariesToPreload(List<string> libraries)
             base.GetLibrariesToPreload(libraries);
         }
 
-        private void SelectAll()
+        private void SelectAll(IEnumerable<Guid> nodes)
         {
             DynamoSelection.Instance.ClearSelection();
             foreach (var node in CurrentDynamoModel.CurrentWorkspace.Nodes)
             {
-                DynamoSelection.Instance.Selection.Add(node); 
+                if (nodes.Contains(node.GUID))
+                    DynamoSelection.Instance.Selection.Add(node); 
```

Comment:
Just for fun, [in go curly braces can't be on the newline](http://stackoverflow.com/questions/17153838/why-does-golang-enforce-curly-bracket-to-not-be-on-the-next-line) :-)